src/wallet/wallet.cpp

```diff
@@ -5120,12 +5118,23 @@ CWallet* CWallet::CreateWalletFromFile(const std::string& name, const fs::path&
     // wallet creation fails.
     auto temp_wallet = MakeUnique<CWallet>(name, WalletDatabase::Create(path));
     CWallet *walletInstance = temp_wallet.get();
-    DBErrors nLoadWalletRet = walletInstance->LoadWallet(fFirstRun);
+    AddWallet(walletInstance);
```

Comment:
and below

Reply:
Those are to make sure there is always consistency between `vpwallets` and `privateSendClientManagers`. This is required because `interfaces::CPrivateSend::Client` is member of `interfaces::Wallet` and therefor requires to have the wallet already in `privateSendClientManagers` before the first `interfaces::Wallet` instance gets created which happens during wallet loading when `uiInterface.LoadWallet(walletInstance)` is called.